src/Withdrawal.java

```diff
@@ -116,9 +116,14 @@ private int displayMenuOfAmounts()
             case 2: // (i.e., chose option 1, 2, 3, 4 or 5), return the
             case 3: // corresponding amount from amounts array
             case 4:
-            case 5:
                userChoice = amounts[ input ]; // save user's choice
                break;       
+            case 5:
+               screen.displayMessage( "\nHow much do you want to withdraw? " );
+               int customAmount = keypad.getInput(); // get user input through keypad
```

Comment:
Do we need to add a check on the type of the inserted input to guard against possible runtime exception?

Reply:
Good catch.
Currently all our ATM have only numeric pad, so it's impossible to insert a letter.
I create a SPIKE to analyse if there's the possibility/need.